### PR TITLE
[armnn] object classification unittest + bug solve

### PR DESCRIPTION
**Pull request by kparichay**
- Add object classification unittest using tflite quantized mobilenet
- Add orange.png raw format image orange.raw
- As armnn treats uint8 datatype of tflite as quant asymm8 (arc/armnnTfLiteParser/TfLiteParser.cpp:310), quant asymm8 is interpreted as uint8 in nnstreamer
- Updated current default HW backend to CpuAcc (based on neon) as CpuRef takes over
5 min to complete the mobilenet unittest (backend h/w configutation will be done properly after #1963 is merged)

Related Issue: #1817

Self evaluation:
Build test: [x]Passed [ ]Failed [ ]Skipped
Run test: [x]Passed [ ]Failed [ ]Skipped

Bug solve: The dimension of data received by armnn are inversed to match the desired order

Signed-off-by: Parichay Kapoor

### CODE REVIEW

File: tests/nnstreamer_filter_armnn/unittest_filter_armnn.cpp [hunk 1]
```diff
@@ -218,6 +222,115 @@ TEST (nnstreamer_filter_armnn, invoke_basic)
...
+  input.data = g_malloc(input.size);
+  char *data_file = g_build_filename (root_path, "tests", "test_models", "data",
+      "orange.raw", NULL);
+  int fd = open(data_file, O_RDONLY);
+  g_free (data_file);
+  ssize_t data_read = read(fd, input.data, input.size);
+  EXPECT_EQ (data_read, input.size);
+  close(fd);
+
+  output.data = g_malloc(output.size);
```
Comment (dongju-chae) [inline tests/nnstreamer_filter_armnn/unittest_filter_armnn.cpp]: free?

Comment (kparichay) [reply tests/nnstreamer_filter_armnn/unittest_filter_armnn.cpp]: my bad, updated.

File: tests/nnstreamer_filter_armnn/unittest_filter_armnn.cpp [hunk 2]
```diff
@@ -220,6 +224,125 @@ TEST (nnstreamer_filter_armnn, invoke_basic)
...
+/**
+ * @brief get argmax from the array
+ */
+size_t get_argmax (guint8 *array, size_t size)
+{
+  size_t idx, max_idx = 0;
+  guint8 max_value = 0;
+  for (idx = 0; idx < size; idx ++)
+    if (max_value < array[idx])
+      max_idx = idx;
```
Comment (dongju-chae) [inline tests/nnstreamer_filter_armnn/unittest_filter_armnn.cpp]: I think you need to update `max_value` as well.

Comment (kparichay) [reply tests/nnstreamer_filter_armnn/unittest_filter_armnn.cpp]: Oh, I wonder how this unit-test worked without this.
Updated

Comment (dongju-chae) [reply tests/nnstreamer_filter_armnn/unittest_filter_armnn.cpp]: Amazing :)

**Pull request merged**